### ISSUE TITLE
nz-alert left padding default value changed from 24px to 63px when using nzDescription

### ISSUE THREAD

**Issue by tihomirleka** [NG-ZORRO/ng-zorro-antd#6220]
### Reproduction link
[https://stackblitz.com/run](https://stackblitz.com/run)

### Steps to reproduce
```
<nz-alert nzType="success" nzMessage="Success Tips" nzShowIcon></nz-alert>
<nz-alert
    nzType="success"
    nzMessage="Success Tips"
    nzDescription="Detailed description and advices about successful copywriting."
    nzShowIcon>
</nz-alert>
```

### What is expected?
Both <nz-alert> elements should contain left padding of 24

### What is actually happening?
When nzDesription is set left padding becames 63px

| Environment | Info |
|---|---|
| ng-zorro-antd | 10.2.2 |
| Browser | Chrome |

---
In the documentation (https://ng.ant.design/components/alert/en#ng-content) component renders ok and left padding is 24px but if you take it out of that context it renders as 63px.

**Comment by stygian-desolator** [comment]
https://stackblitz.com/edit/angular-396brq?file=src/app/app.component.ts

It seems to be the same as in the official document 🤔

**Comment by tihomirleka** [comment]
My bad, reproduction link was wrong. Here you should be able to see the problem.

https://stackblitz.com/edit/angular-396brq-gqt5s2?file=src/app/app.component.ts

**Comment by wenqi73** [comment]
Has fixed in 11.0.0.

**Comment by jarkym** [comment]
Still 63px on version 11.3
